scripts/install.js

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
+
+var log = require('npmlog');
+var execSync = require('child_process').execSync;
+
+// Default command
+var fetchCommand = 'node-pre-gyp install --fallback-to-build=false';
+var buildCommand = 'cmake-js compile';
+
+// ==== Determine install options ====
+// Skip the fetch stage. '--no-fetch', or '--fetch=false'
+var skipFetch = process.env.hasOwnProperty('npm_config_fetch') && !process.env['npm_config_fetch'];
```

Comment:
>'npm_config_fetch' [](start = 43, length = 18)

Please explain where the transformation between --no-fetch and napa_config_fetch happens.